src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -1995,6 +1995,9 @@
   <data name="ERR_TypeRefResolutionError3" xml:space="preserve">
     <value>Import of type '{0}' from assembly or module '{1}' failed.</value>
   </data>
+  <data name="ERR_ValueTupleTypeRefResolutionError" xml:space="preserve">
+    <value>Import of type '{0}' failed.</value>
```

Comment:
New message is like C#:
"BC37267: Predefined type 'ValueTuple(Of ,)' is not defined or imported."